lib/modules/manager/kotlin-script/readme.md

```diff
@@ -1,11 +1,6 @@
----
-title: Kotlin Script dependency versions
-description: Kotlin Script dependency versions support in Renovate
----
-
 Renovate supports upgrading dependencies in [Kotlin Script](https://github.com/Kotlin/KEEP/blob/master/proposals/scripting-support.md) files.
-These are self-contained scripts where one can write Kotlin code with JVM backend, and compilation happens when the
-scripts are ran. For example:
+These are self-contained scripts where one can write Kotlin code with JVM backend, and compilation happens when the scripts are ran.
```

Comment:
```suggestion
You can use Kotlin Script to write Kotlin code with the JVM backend.
Compilation happens when the Kotlin Script runs.
```

The current sentence feels ungrammatical. How about this? Also should it be `with the JVM backend` or `on the JVM backend`?

Reply:
I think this sentence can be removed as it isn't really relevant to Renovate.

Reply:
I agree regarding the removal, but I think we should keep the current link or a similar one e.g. https://kotlinlang.org/docs/custom-script-deps-tutorial.html

Reply:
```suggestion
Read the [Kotlin Script docs](https://kotlinlang.org/docs/custom-script-deps-tutorial.html) to learn more.
```